website/docs/r/auditing.html.markdown

```diff
@@ -0,0 +1,47 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: auditing"
+sidebar_current: "docs-mongodbatlas-resource-auditing"
+description: |-
+    Provides a Auditing resource.
+---
+
+# mongodbatlas_auditing
+
+`mongodbatlas_auditing` provides a Auditing resource. This allows auditing to be created.
+
+## Example Usage
+
+```hcl
+resource "mongodbatlas_auditing" "test" {
+		project_id                  = "<project-id>"
+		audit_filter                = "{ 'atype': 'authenticate', 'param': {   'user': 'auditAdmin',   'db': 'admin',   'mechanism': 'SCRAM-SHA-1' }}"
+		audit_authorization_success = false
+		enabled                     = true
+	}
+```
+
+## Argument Reference
+
+* `project_id` - (Required) The unique ID for the project to create the database user.
+* `audit_filter` - Indicates whether the auditing system captures successful authentication attempts for audit filters using the "atype" : "authCheck" auditing event. For more information, see auditAuthorizationSuccess
```

Comment:
Can we actually link `auditAuthorizationSuccess` to the appropriate MongoDB docs page? By itself, it doesn't make much sense here.

Reply:
```suggestion
* `audit_filter` - Indicates whether the auditing system captures successful authentication attempts for audit filters using the "atype" : "authCheck" auditing event. For more information, see [auditAuthorizationSuccess](https://docs.mongodb.com/manual/reference/parameters/#param.auditAuthorizationSuccess).
```